utils/cp_utils.py

```diff
@@ -66,9 +66,8 @@ def run_cellprofiler(
         # A log file is created for each plate or data set name (based on folder name with images)
         with open(pathlib.Path(f"logs/cellprofiler_output_{pathlib.Path(path_to_images).name}.log"), "w") as cellprofiler_output_file:
             # run CellProfiler for a illumination correction pipeline
-            command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
-            subprocess.run(command, shell=True, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file)
-        cellprofiler_output_file.close()
+            command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}".split()
```

Comment:
Consider transitioning this (and the other) command string to a list, avoiding the split and f-string notation. If you do keep this format instead, consider providing a comment to note why the `shell` parameter is not used.

Reply:
Added in https://github.com/WayScience/nf1_cellpainting_data/pull/5/commits/e5195b9daf9af364ce74422ebe7d48a6568fe69f